Notifications test not working

## Expected Behavior

Running "docker run -it --rm --env-file ./dotenv ghcr.io/jef/streetmerchant:latest test:notification:production" should send a test notification

## Current Behavior

An error is thrown

## Steps to Reproduce

Run "docker run -it --rm --env-file ./dotenv ghcr.io/jef/streetmerchant:latest test:notification:production"

## Environment

- OS: Docker

## Logs

> streetmerchant@ test:notification:production /app
> node build/test/functional/test-notification.js

internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module '/app/build/test/functional/test-notification.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! streetmerchant@ test:notification:production: `node build/test/functional/test-notification.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the streetmerchant@ test:notification:production script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/appuser/.npm/_logs/2021-01-18T21_11_14_742Z-debug.log

--- a/Dockerfile
+++ b/Dockerfile
@@ -12,6 +12,7 @@ COPY tsconfig.json tsconfig.json
 RUN npm ci
 
 COPY src/ src/
+COPY test/ test/
 RUN npm run compile
 RUN npm prune --production
 

